crates/client/assets/schema.sdl

```diff
@@ -198,6 +198,7 @@ type ConsensusParameters {
 	feeParams: FeeParameters!
 	baseAssetId: AssetId!
 	blockGasLimit: U64!
+	blockTransactionSizeLimit: U64!
```

Comment:
When I first read it, I understood that it was the maximum size for a transaction. I think I misunderstood because the word transaction is singular. IMO, it should be plural.

Reply:
Yes, the name could indeed be slightly improved. However, it's already merged in `fuel-vm` [here](https://github.com/FuelLabs/fuel-vm/pull/821/files). 

In the follow-up PR to `fuel-core` I'll make sure that this is comprehensively documented.